### ISSUE TITLE
Failed Task Metrics

### ISSUE THREAD

**Issue by rwalshe** [astronomer/telescope#197]
It would be helpful to see the count of monthly failed tasks in the Telescope report.

**Comment by fritz-astronomer** [comment]
@rwalshe I think this exists in source data, so likely a small fix to propagate it through to sigma (which, I assume, is what you mean)

**Comment by rwalshe** [comment]
@fritz-astronomer correct, visibility in the Sigma Dashboard presenting the failed task count.  If there was further granularity available in the source data on tasks states, this would be great to see, though the ask is simply success/failure visibility.
